kong/plugins/httplog/schema.lua

```diff
@@ -0,0 +1,6 @@
+return {
+  logging_url = { required = true, type = "string" },
+  method = { require = false, default = "POST", enum = { "POST", "PUT", "PATCH" } },
```

Comment:
Also it's `required`, there is a typo here. (And require is false by default so actually no need.